lib/alchemy/on_page_layout.rb

```diff
@@ -33,11 +33,12 @@ module Alchemy
   #     end
   #
   module OnPageLayout
-    mattr_accessor :callbacks
+    mattr_accessor :callbacks do
+      {}
```

Comment:
Nice, didn't know you can define defaults. 

But, I would like to use a shorter syntax like:

```
mattr_accessor(:callbacks) { Hash.new }
```